packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -65,7 +65,10 @@ export interface IOdspResponse<T> {
 	duration: number;
 }
 
-export interface TokenFetchOptionsEx extends TokenFetchOptions {
+export interface TokenFetchOptionsEx {
```

Comment:
Can you please add some comments on where it's used, and its relationship to TokenFetchOptions ?